Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb

```diff
@@ -0,0 +1 @@
+$("<%= escape_javascript(render @note) %>").appendTo("#notes");
```

Comment:
well, in this case, I think it grabs _note.html.erb, renders that into HTML, then appends that html to the #notes div. That appears to me to be how it's working from what I could glean from documentation and random people on the internet.